src/frontend/apps/impress/src/features/docs/doc-tree/components/DocSubPageItem.tsx

```diff
@@ -76,62 +87,72 @@ export const DocSubPageItem = (props: TreeViewNodeProps<Doc>) => {
     }
   };
 
+  useKeyboardActivation(
```

Comment:
When I click on "Enter" it does not seem to  fold / unfold, with "space" yes though.